app/views/browse/_pod-details.html

```diff
@@ -25,6 +25,7 @@
           <a ng-href="{{rcName | navigateResourceURL : 'ReplicationController' : pod.metadata.namespace}}"
             ><span ng-if="deploymentVersion">#{{deploymentVersion}}</span><span ng-if="!deploymentVersion">{{rcName}}</span></a></span>
         </dd>
+        <owner-references ng-if="!dcName" api-object="pod"></owner-references>
```

Comment:
@jwforres Drawback here is that if the pod has other owner references besides the rc, they won't show up. But switching to always use `<owner-references>` instead of the special handling above changes

```
Deployment: mongodb, #1
```

to

```
Replication Controller: mongodb-1
```

Can't decide what's best.